cmd/run.go

```diff
@@ -461,6 +465,13 @@ func RunServer(ctx context.Context, config *Config) error {
 		}()
 	}
 
+	logger.Info(fmt.Sprintf("🧪 experimental features enabled: %v", config.ExperimentalsEnabled))
+
+	var experimentals []server.ExperimentalFeatureFlag
+	for _, feature := range config.ExperimentalsEnabled {
```

Comment:
I looked into implementing the pflag interface for the custom `ExperimentalFeatureFlag` type and for a `[]ExperimentalFeatureFlag` it looked more complicated than it was worth just for converting a slice of strings to a custom type. So I chose this route instead.

I may revisit that here shortly and find a better way, but this is simple and straightforward and works just fine to get this functionality out.